src/components/Chatbot/Chatbot.tsx

```diff
@@ -23,46 +19,115 @@ const NewConversationButton = styled(PrimaryButton)(() => ({
 }));
 
 export const Chatbot = () => {
-  const [conversation, setConversation] = useState<Message[]>([]);
+  const [conversation, setConversation] = useState<MessageBubbleProps[]>([]);
   const [error, setError] = useState(false);
   const [loading, setLoading] = useState(false);
+  const [conversationCreated, setConversationCreated] = useState<string | null>(
+    null
+  );
+
+  const loadConversationFromBackend = async () => {
+    try {
+      await loadConversation().then((response) => {
+        if (response.ok) {
+          setConversation(response.conversation.messages);
+          setConversationCreated(response.conversation.created_at.slice(0, 10));
```

Comment:
This is not the optimal time formatting 😄

Reply:
what is the optimal time formatting?

Reply:
For example `(new Date(response.conversation.created_at)).toLocaleDateString()`

Reply:
`created_at` is an ISO string

Reply:
This does not seem to be resolved.